acceptance/single_key_test.go

```diff
@@ -95,7 +95,7 @@ func TestSingleKey(t *testing.T) {
 					if err := v.UnmarshalBinary(r.ValueBytes()); err != nil {
 						return err
 					}
-					return txn.Commit(txn.B.Put(key, v+1))
+					return txn.Commit(client.B().Put(key, v+1))
```

Comment:
`client.B()` is 10 characters and yet still fairly cryptic. If the invocation is going to be this long can we call it `Batch()` instead of `B()`? (or maybe do both: name the real function `Batch()` for better documentation and define `B()` as an alias for it).  

Adding a Batch() method to Txn and DB would also be a way to save a few characters, although it would falsely imply that batches are related to the Txn/DB from which they were created.

Also note that to us it's package `client`, but to the outside world `cockroach` is a better package name and we might want to rename it accordingly some day. This will make static functions more verbose.